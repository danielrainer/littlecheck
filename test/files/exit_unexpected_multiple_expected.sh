# RUN: /bin/sh %s
# CHECKEXIT: 1 SIGINT -100
# CHECKEXIT: KILL -2
