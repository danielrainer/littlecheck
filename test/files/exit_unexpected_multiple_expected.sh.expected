Failure in exit_unexpected_multiple_expected.sh:

  Process unexpectedly exited with code 0 (expected: -100 or 1 or SIGINT (Interrupt) or SIGKILL (Killed))

  when running command:
    /bin/sh exit_unexpected_multiple_expected.sh
